doc/manuals/admin/perf_tuning.md

```diff
@@ -14,12 +14,12 @@
 
 ##  MongoDB configuration
 
-Since version 0.21.0, Orion supports both MongoDB 2.6 and MongoDB 3.0 without difference from a functional
-point of view. However, MongoDB 2.6 implements a per-collection lock, while MongoDB 3.0 (when configured
-to use WireTiger storage engine) implements per-document lock. Thus, the lock system in MongoDB 3.0
+Since version 0.21.0, Orion supports both MongoDB 2.6, 3.0 and 3.2 without difference from a functional
```

Comment:
both ...
Since version 0.21.0, Orion supports MongoDB versions 2.6, 3.0 and 3.2 without difference ...